src/content/docs/journey-migration/migrating-dashboards.mdx

```diff
@@ -57,32 +57,32 @@ Our dashboards take the data you put into New Relic and provide you with a wide
+        Go to [**one.newrelic.com > Add Data**](https://one.newrelic.com/marketplace), then select an integration. For this example, we'll choose **Infrastructure & OS**, and create a dashboard to monitor an Azure integration.  
         </Step>
         <Step>
         Select the **Azure Integrations Data Ingest Analysis** option under **Dashboards** to open the guided install window.
         </Step>
         <Step>
-        Select the account you want to add the dashboard to from the drop-down.
+        From the drop-down, select an account.
         </Step>
         <Step>
-        Ensure you have the data stream setup already. In this case, an infrastructure monitor would need to be setup. If you hadn't set up, you'd select the **Setup** option, and if you had, you'd select **Done**.
+        If you haven't installed your integration yet, choose **Setup**. If you installed already, choose **Done**.
         </Step>
         <Step>
         New Relic creates the dashboard, and you can select **View dashboard** to go to it right away and begin visualizing your data stream.
         </Step>
     </Steps>
-    If you want to learn how to [modify existing dashboard components](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-manage) or [create new ones from scratch](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-create-new), check out our comprehensive [dashboard documentation](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+    If you want to learn how to [modify existing dashboard components](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-manage) or [create new ones from scratch](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-create-new), check out our [dashboard documentation](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
     </Side>
     </SideBySide>
     </Collapser>
 </CollapserGroup>
 
 ## Your alerting solution [#alerts-solutions]
 
-Being able to see your data is great, but being able to act on it to correct or prevent errors is even better! New Relic gives you multiple ways to customize alerts for your unique requirements while helping to eliminate alert fatigue. We offer proactive awareness for your problems, utilizing machine learning and applied intelligence to enhance our policies. Our alerts:
+Being able to see your data is great, but being able to act on it to correct or prevent errors is even better! New Relic gives you multiple ways to customize alerts for your unique requirements while helping to eliminate alert fatigue. An alert will notify you of an issue before it becomes a problem, using machine learning and applied intelligence to enhance our policies. Here's what sets our alerts apart:
```

Comment:
"we give you multiple ways" Again, if you can avoid "New Relic" it's worth doing

Reply:
Fixed, thank you!